policyengine_us/parameters/gov/contrib/congress/golden/fisc_act/family_income_supplement/amount/pregnant.yaml

```diff
@@ -0,0 +1,10 @@
+description: Representative Jared Golden proposed a family income supplement of this amount for pregnant women.
+metadata:
+  unit: currency-USD
+  label: FISC Act family income supplement pregnant amount
+  period: month
+  reference:
+    - title: Family Income Supplemental Credit Act, Sec. 2, (d)(1)
+      href: https://golden.house.gov/sites/evo-subsites/golden.house.gov/files/evo-media-document/GoldenFISC.pdf#page=5
+values:
```

Comment:
how about making this a scale parameter to capture the 20 week requirement?

Reply:
Agreed, we currently model `current_pregnancy_month` we can make the distinction based on this value - to avoid creating more pregnancy related vars for different programs